networking/synthetic/stac/Dockerfile

```diff
@@ -47,21 +47,34 @@ RUN yum install --assumeyes \
 RUN wget ${stac_url}; \
     tar zxf STAC-N-Tools-Pkg20131015_jeder.tar.gz; \
     mv STAC-N-Tools-Pkg20131015 /opt/; \
-    #echo -e "set -x\n$(cat /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration-initial-customization/common_tcp.sh)" > /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration-initial-customization/runTest.sh; \
-    #echo -e "set -x\n$(cat /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration-initial-customization/common_tcp.sh)" > /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration-initial-customization/common_tcp.sh; \
-    sed -i -- "s/ssh/ssh -q/" /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration-initial-customization/runTest.sh; \
-    sed -i -- "s/ssh/ssh -q/" /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration-initial-customization/common_tcp.sh; \
-    sed -i -- "s/scp/scp -q/" /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration-initial-customization/runTest.sh; \
-    sed -i "s/.*RUN_WITH_SCREEN=.*/export RUN_WITH_SCREEN=0/" /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration-initial-customization/run_common.sh; \
-    sed -i "s/producer1=.*/producer1=${producer}/" /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration-initial-customization/run_common.sh; \
-    sed -i "s/consumer1=.*/consumer1=${consumer}/" /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration-initial-customization/run_common.sh; \
-    sed -i -- "s/p1p1/${iface_name}/" /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration-initial-customization/common_tcp.sh; \
-    sed -i  "s/ADAPTER_PRODUCER_LISTEN_ADDR=.*/ADAPTER_PRODUCER_LISTEN_ADDR=${producer}/" /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration-initial-customization/common_tcp.sh; \
-    sed -i  "s/ADAPTER_CONSUMER_LISTEN_ADDR=.*/ADAPTER_CONSUMER_LISTEN_ADDR=${consumer}/" /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration-initial-customization/common_tcp.sh; \
+    #echo -e "set -x\n$(cat /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration/common_tcp.sh)" > /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration/runTest.sh; \
+    #echo -e "set -x\n$(cat /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration/common_tcp.sh)" > /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration/common_tcp.sh; \
+    sed -i -- "s/ssh/ssh -q/" /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration/runTest.sh; \
+    sed -i -- "s/ssh/ssh -q/" /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration/common_tcp.sh; \
+    sed -i -- "s/scp/scp -q/" /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration/runTest.sh; \
+    sed -i -- "s/taskset -c 1,3/taskset -c 2,4/" /opt/STAC-N-Tools-Pkg20131015/Linux/orchestration/common_tcp.sh; \
```

Comment:
This tuning is specific to the machines you ran on.  Just a note -- variablized in ansible this would be very helpful.